edk2toolext/tests/test_edk2_ci_build.py

```diff
@@ -0,0 +1,58 @@
+# @file test_edk2_ci_build.py
+# This contains unit tests for the edk2_ci_build
+##
+# Copyright (c) Microsoft Corporation
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+import unittest
+from edk2toolext.invocables.edk2_ci_build import Edk2CiBuild
+import sys
+import os
+import logging
+import shutil
+from importlib import reload
+from edk2toolext.environment import shell_environment
+
+
+class TestEdk2CiBuild(unittest.TestCase):
+
+    minimalTree = os.path.join(os.path.dirname(__file__), "minimal_uefi_tree")
+
+    def setUp(self):
+        TestEdk2CiBuild.restart_logging()
```

Comment:
What's the significance of all these restarts?

Reply:
Comment added